include/util/coordinate.hpp

```diff
@@ -40,7 +40,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace osrm
 {
 
-constexpr const double COORDINATE_PRECISION = 1e6;
+constexpr const double COORDINATE_PRECISION = 1e7;
```

Comment:
We still include internal headers in our public interface - this exposes internals such as the coordinate precision, too. Check this:

https://github.com/Project-OSRM/osrm-backend/blob/fce3bb180c56d0fc70d439c8e943cd6a3003363b/include/util/coordinate.hpp#L40-L43

osrm namespace, exposed via the `osrm/coordinate.hpp` hack here:

https://github.com/Project-OSRM/osrm-backend/blob/1ae69fe40d68a15ca978f0a4535d75523b292467/include/osrm/coordinate.hpp#L31

This is ticketed since some time now in https://github.com/Project-OSRM/osrm-backend/issues/2031.
If we're fine with this kind of breakage, I'm happy to include the changeset.